crashtracker/src/telemetry.rs

```diff
@@ -70,7 +70,7 @@ impl TelemetryCrashUploader {
 
             // ignore result because what are we going to do?
             let _ = if endpoint.url.scheme_str() == Some("file") {
-                cfg.set_url(&format!("file://{}.telemetry", endpoint.url.path()))
+                cfg.set_host(&format!("file://{}.telemetry", endpoint.url.path()))
```

Comment:
I think `set_host` is clearer than`set_url` (I wrote the method and I ended up confused on what it did exactly, and naming wasn't a great help), because this method just sets the host and discards any path on the string

Reply:
`set_host_from_url` ?

Reply:
I still feel that `set_host...` implies that there should be a `set_port...`, but `set_host_from_url` is fine.